### DISCUSSION TITLE
建议商城中增加单独购买隧道数量的服务

### DISCUSSION THREAD

**Discussion by chwncy** [natfrp/wiki#92] [想法] [answered]
普通用户对每月赠送的流量完全够用，但隧道数量却只有两条，而vip中的额外流量却没有必要，建议增加单独购买隧道数量的服务来弥补一部分用户对隧道的需求。

**Comment by fakeboboliu** [comment, marked as answer]
目前来说，我们的普通节点负载情况并不能支持免费用户使用更多的隧道。
在目前我们要实现收支平衡的原则下，开放隧道数购买意味着这一项的收入要用来平衡添加普通节点的成本，显然其价格对于目前有过类似问题的用户规模来说将是无法接受的。
